cluster/saltbase/salt/kube-proxy/kube-proxy.manifest

```diff
@@ -29,8 +29,10 @@
  {% set feature_gates = "--feature-gates=" + grains.feature_gates -%}
 {% endif -%}
 
+{% set throttles = "--iptables-sync-period=1m --iptables-min-sync-period=10s" -%}
```

Comment:
we can make this even more complex and add a short holdoff (e.g. 1sec).  I'd rather do that as a followup, if needed.  Or simpler, make bursts higher.  Here's my thinking - if the iptables update is fast, a short burst isn't a big deal.  If the update is slow, there will be a lot of time between runs anyway to accumulate new pods.  So maybe setting burst to 4 or is better - we could do it as a follow up to make that a flag...